Add license keys for Bloom and GDS to Additional files to back up (#1540)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/planning.adoc

The locations of these are described in xref:security/ssl-framework.adoc[SSL framework].
If you have a cluster, you should back up these files for each cluster member.
* If using custom plugins, make sure that you have the plugins in a safe location.

> * If using Bloom or GDS Enterprise, license key files for these products should be backed up as well.

Reviewer: I have only one question: which modal verb to use: _should_ or _have_? 
```suggestion
* If using Bloom or GDS Enterprise, you have to back up license key files for these products as well.
```
